data/John Verhoefen (a commercial agent).txt

```diff
@@ -75,6 +75,7 @@ mission "John Verhoefen: First Contact"
 	to offer
 		random < 20
 		"reputation: Merchant" >= 10
+		"John Verhoefen: stage" == 0
```

Comment:
The mission does not use `repeat` by purpose and thus I did not implement additional checks since I do not use any `defer`